src/components/Properties.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+function Properties() {
+  return <h1>Properties Page</h1>;
+}
+export default Properties;
```

Comment:
Not much in the component I know, but always good practice to leave a clear line before the `export` statement at the end